spec/deploy_spec.rb

```diff
@@ -165,63 +165,6 @@
             end
           end
         end
-
```

Comment:
Yes, these tests will need to be updated, given the changes to the Deploy_Controller